app/controllers/assets_controller.rb

```diff
@@ -155,6 +155,13 @@ def view
     render layout: false
   end
 
+  def start_edit
```

Comment:
This should be a more descriptive name, like `create_start_edit_image_activity`or along those lines, it's too generic here.